presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfigurationInitializer.java

```diff
@@ -69,17 +74,20 @@ public RubixConfigurationInitializer(RubixConfig config)
     }
 
     @Override
-    public void initializeConfiguration(Configuration config)
+    public void updateConfiguration(Configuration config, HdfsContext context, URI uri)
     {
+        checkState(getCacheKey(config) == null, "Rubix does not support dynamic FileSystems");
```

Comment:
Are we in control over ordering of DynamicConfigurationProviders. This check makes sense only if we know RubixConfigurationInitializer will run last. Right?